### PR TITLE
Fixed merge conflict from PR #14 and added console logging

### PR DESCRIPTION
**Pull request by dangothemango**
I added a few messages in app.js just so that the program would give output on startup to make sure it worked. Dependency for i and arp-listener removed. and updated dependency for node-dash-button.

### CODE REVIEW

**Comment by mynamebrody** [comment]
👍 LGTM. @JoeIaquinto?

**Comment by JoeIaquinto** [comment]
All good changes. Should Dan put himself in contributors?

**Comment by mynamebrody** [comment]
Yeah, @dangothemango you can go ahead and do that as well

**Comment by dangothemango** [comment]
alright done

File: package.json [hunk 1]
```diff
@@ -44,6 +44,11 @@
       "name": "Joseph Iaquinto",
       "email": "iaquinto.joe@gmail.com",
       "url": "http://joeiaquinto.github.io/"
+    },
+    {
+      "name": "Dan Gorman",
+      "email": "danthemangorman@gmail.com"
```
Comment (mynamebrody) [inline package.json]: Missing comma here

**Comment by dangothemango** [comment]
my bad

**Pull request merged**